black.py

```diff
@@ -3493,20 +3494,32 @@ def __str__(self) -> str:
         return ", ".join(report) + "."
 
 
-def parse_ast(src: str) -> Union[ast3.AST, ast27.AST]:
-    for feature_version in (7, 6):
-        try:
-            return ast3.parse(src, feature_version=feature_version)
-        except SyntaxError:
-            continue
+def parse_ast(src: str) -> Union[ast.AST, ast3.AST, ast27.AST]:
+    major, minor = sys.version_info[:2]
+    # TODO: support Python4+ ;)
+    if major == 3 and minor >= 8:
```

Comment:
You can just do `if (major, minor) >= (3, 8)` and we'll have Python 4 support :)

Reply:
No because the for loop below :(